### ISSUE TITLE
feat: Don't compact block scalars ( > ) into one line

### ISSUE THREAD

**Issue by yoskeoka** [google/yamlfmt#63]
I would like to keep newlines in block scalars for better readability.
But yamlfmt (v0.5.0)'s basic formatter changes block scalar with `>` into one line.

e.g.

```yaml
commands: >
  [ -f "/usr/local/bin/foo" ] &&
  echo "skip install" ||
  go install github.com/foo/foo@latest
```

yamlfmt make this file like below

```yaml
commands: >
  [ -f "/usr/local/bin/foo" ] && echo "skip install" || go install github.com/foo/foo@latest
```

**Comment by braydonk** [comment]
Thanks for making an issue! 

Yeah, that sucks. Similarly to #39 this is going to be really hard to deal with since it's in the `yaml.v3` library. However I have started the process of forking that library so I can actually patch it effectively, so I can probably tackle this once I've done that.

**Comment by braydonk** [comment]
This fix is in `main` and will be in `v0.7.0`!
